R/calibrate.R

```diff
@@ -460,7 +460,8 @@ calibrate_outputs <- function(output,
              mode = mode * ratio,
              lower = lower * ratio,
              upper = upper * ratio,
-             ratio = NULL
+             ratio = NULL,
+             data_type = "model_estimate"
```

Comment:
Can you remind me where `data_type` is used?

Reply:
We discussed this over teams, was left in by mistake and has since been removed.